### PR TITLE
Remove redundant opt func tests

### PR DESCRIPTION
**Pull request by hlinnaka**

### CODE REVIEW

File: src/test/tinc/tincrepo/dml/functional/expected_partition/mpp21090_drop_firstcol_dml_int_ao.ans [hunk 1]
```diff
@@ -1,47 +0,0 @@
-DROP TABLE IF EXISTS mpp21090_drop_firstcol_dml_int;
-psql:/data/prabhd/test/tincrepo/main/dml/functional/output/mpp21090_drop_firstcol_dml_int_ao_opt.sql:14: NOTICE:  table "mpp21090_drop_firstcol_dml_int" does not exist, skipping
```
Comment (vraghavan78) [inline src/test/tinc/tincrepo/dml/functional/expected_partition/mpp21090_drop_firstcol_dml_int_ao.ans]: can we keep this? There was a bug and this test was added.

Comment (hlinnaka) [inline src/test/tinc/tincrepo/dml/functional/expected_partition/mpp21090_drop_firstcol_dml_int_ao.ans]: Did you read the explanations in the commit messages?

Comment (hlinnaka) [inline src/test/tinc/tincrepo/dml/functional/expected_partition/mpp21090_drop_firstcol_dml_int_ao.ans]: FYI, I plan to submit more patches to remove the bulk of those mpp21090 test. I can see the need for testing for that - it was a subtle bug - but the amount of tests we have for that now seems completely overblown. But even if you disagree with that, these "int" tests being removed in this PR are clearly redundant with the int4 tests.

Comment (vraghavan78) [inline src/test/tinc/tincrepo/dml/functional/expected_partition/mpp21090_drop_firstcol_dml_int_ao.ans]: Sorry. Let me take a look.

**Comment by hlinnaka** [comment]
Ok, pushed, thanks. Sounds like we will have a fight ahead on the remaining mpp21090 tests ;-)

**Pull request closed without merging**